test/ctia/test_helpers/aggregate.clj

```diff
@@ -126,6 +132,7 @@
           res-days (map #(to-granularity-first-day granularity %)
                         date-values)
           expected (make-histogram-res res-days)
+          _ (every? #(-> % first :value) expected)
```

Comment:
missing `(assert` I believe

Reply:
Indeed :-)! I guess that time is running out to wake up today ;-)